v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/WriteToElasticsearch.java

```diff
@@ -66,14 +63,19 @@ public static Builder newBuilder() {
 
   public abstract ElasticsearchWriteOptions options();
 
+  private static final String DOCUMENT_TYPE="_doc";
```

Comment:
Can you add a comment saying "Types have been removed in ES 7.0. Default will be _doc. See https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html"